src/guide/testing-the-jdk.md

```diff
@@ -167,6 +169,16 @@ make test TEST=gtest:$X/$variant
 
 The second example above runs tests which match the regexp `$X.*` on a specific variant of the JVM. The variant is one of client, server, etc.
 
+## GitHub actions
+
+GitHub has a feature called **GitHub actions** (GHA) that can be used to automate testing. The GHA is executed whenever a push is made to a branch in your repository. The bots will show the results of the GHA in your PR when you create or update it. The GHA in the JDK project is configured to run a set of tests that is commonly known as **tier 1**. This is a relatively fast, small set of tests that tries to verify that your change didn't break the JDK completely. In tier 1 the JDK is built on a small set of platforms including (but not necessarily limited to) Linux, Windows, and MacOS, and a few tests are executed using these builds.
```

Comment:
GHA also does a whole bunch of cross-compilation builds for PPC, ARM and S390.

Reply:
Added a sentence about this.